nvi-evaluator/src/test/java/no/sikt/nva/nvi/evaluator/EvaluateNviCandidateHandlerTest.java

```diff
@@ -343,6 +343,17 @@ void shouldCreateNonCandidateEventWhenIdentityIsNotVerified() throws IOException
         assertThat(candidate.candidate().getClass(), is(equalTo(NonNviCandidate.class)));
     }
 
+    @Test
+    void shouldCreateNonCandidateWhenPublicationYearBefore2022() throws IOException {
+        var path = "candidate_publicationDate_before_2022.json";
+        var content = IoUtils.inputStreamFromResources(path);
+        var fileUri = s3Driver.insertFile(UnixPath.of(path), content);
+        var event = createS3Event(fileUri);
+        handler.handleRequest(event, output, context);
+        var candidate = getMessageBody();
+        assertThat(candidate.candidate().getClass(), is(equalTo(NonNviCandidate.class)));
+    }
```

Comment:
Gjelder dette kun importen? Eller alle poster som skal publiserer i NVA for fjoråret?

Reply:
Det vil påvirke alle poster, men er satt på for å håndtere importen til TEST nå. Frem til vi finner en løsning for bath indeksering for nvi e.l.

Reply:
Vi bør notere dette ned :)